packages/tooling/__tests__/lib/parameters-to-json-schema.test.js

```diff
@@ -559,11 +559,30 @@ describe('type', () => {
       ];
 
       expect(parametersToJsonSchema({ parameters })[0].schema).toStrictEqual({
-        properties: { param: { type: 'array' } },
+        properties: { param: { type: 'object' } },
```

Comment:
Is this a major breaking change? Or was this test case always broken?

Reply:
Just added a comment on this. It was a broken test case.